types/registry.go

```diff
@@ -0,0 +1,66 @@
+package types
+
+import "time"
+
+// CompanyAddress represents a company address
+type CompanyAddress struct {
+	AdressLine1 string `json:"addressLine1"`
+	AdressLine2 string `json:"addressLine2"`
+	AdressLine3 string `json:"addressLine3"`
+	AdressLine4 string `json:"addressLine4"`
+	PostCode    string `json:"postCode"`
+	Country     string `json:"country"`
+	CareOf      string `json:"careOf"`
+	Type        string `json:"type"`
+}
+
+// CompanyAnnualReturn represents a company's annual return details
+type CompanyAnnualReturn struct {
+	FilingMonth int       `json:"filingMonth"`
+	LastFiled   time.Time `json:"lastFiled"`
+}
+
+// CompanyDirector represents a company director
+type CompanyDirector struct {
+	Name            string    `json:"name"`
+	AppointmentDate time.Time `json:"appointmentDate"`
+	Status          string    `json:"status"`
+}
+
+// ShareAllocation represents a company share allocation
+type ShareAllocation struct {
+	Type       string `json:"type"`
+	Allocation int    `json:"allocation"`
+	Name       string `json:"shareholderName"`
+}
+
+// IndustryClassification represents the company's industry classification
+type IndustryClassification struct {
```

Comment:
We should prefix all these types with `Company` to disambiguate with future types that might have similar names.  `IndustryClassification` should be `CompanyIndustryClassification` or just `CompanyIndustry`

Reply:
Renamed type to `CompanyIndustry`